docs/src/remap/remap.adoc

```diff
@@ -1102,35 +1102,30 @@ def prepare_epilog(self, **words):
 
 ---------------------------------------------------------------------
 
-prepare_prolog and prepare_epilog are part of the 'standard glue'
+The functions 'prepare_prolog' and 'prepare_epilog' are part of the _standard glue_
```

Comment:
```suggestion
The functions _prepare_prolog_ and _prepare_epilog_ are part of the _standard glue_
```
make it future proof and consistent

Reply:
So was I and I prefer my version. prepare_prolog is just a reference to a function that has that name, so it is ok to quote in single quotes.  The standard glue however is more of a concept and thus not quoted but emphasized.